docs/pages/blog/2019.md

```diff
@@ -16,7 +16,7 @@ It puts us on an exciting path to solve even greater challenges in the coming ye
 - ⭐️ From 43.1k to 53.3k stars, leave us [yours 🌟](https://github.com/mui-org/material-ui).
 - 👨‍👩‍👧‍👦 From 1,064 to [1,581](https://github.com/mui-org/material-ui/graphs/contributors) contributors.
 - 💰 Grew financial support by 76% in 2019, compared to 2018.
-- 🏢 From 1.5 to 3 full-time equivalent developers, spread among multiple financially supported [core team members](https://material-ui.com/discover-more/team/).
+- 🏢 From 1.5 to 3 full-time equivalent developers, spread among multiple financially supported [core team members](https://v4.material-ui.com/discover-more/team/).
```

Comment:
I think that it would work too, with not having to hard code the domain

```suggestion
- 🏢 From 1.5 to 3 full-time equivalent developers, spread among multiple financially supported [core team members](/discover-more/team/).
```